C-Unit1-assignments/src/Assignment_02.cpp

```diff
@@ -7,34 +7,40 @@
 // factorialDigits[158] - digits of n!
 //
 
+int carryHandle(int factorialDigits[], int carry, int acount)
+{
+	while (carry)
+	{
+		factorialDigits[acount++] = carry % 10;
+		carry = carry / 10;
+	}
+	return acount;
+}
+int multiply(int factorialDigits[], int acount, int i)
+{
+	int count,j = 0;
+	int carry = 0;
+
+	for (count = 0; count < acount; count++)
+	{
+		int prod = factorialDigits[count] * i + carry;
+		factorialDigits[count] = prod % 10;
+		if (prod >= 10)
+			carry = prod / 10;
+		else
+			carry = 0;
+	}
+	return carryHandle(factorialDigits, carry, acount);
+}
```

Comment:
Good one. This looks neat and understandable function names. Keep this going in the next assignments too 😄